src/sentry/api/endpoints/project_rule_preview.py

```diff
@@ -68,5 +69,11 @@ def post(self, request: Request, project) -> Response:
             on_results=lambda x: serialize(x, request.user),
             count_hits=True,
         )
+
+        inbox_details = get_inbox_details([Group(id=int(g["id"])) for g in response.data])
+        for group in response.data:
+            if int(group["id"]) in inbox_details:
```

Comment:
do we need this conditional check? or can we use logic more similar to this:
https://github.com/getsentry/sentry/blob/ad99c1fe7d7378c3761bad757aacf27352c693a6/src/sentry/api/serializers/models/group_stream.py#L277